jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -40,8 +40,11 @@
      (ros::warn ":angle-vector tm is not a number, use :angle-vector av tm args"))
    (let ((use-torso t))
      (if (and (member :use-torso args) (null (cadr (member :use-torso args)))) (setq use-torso nil))
-     (send self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t)
+     (send* self :angle-vector-motion-plan av :move-arm :rarm :total-time tm :start-offset-time 0.01 :use-torso use-torso :clear-velocities t args)
    ))
+  (:rarm-angle-vector 
+   (av &optional (tm 3000) &rest args)
+   (send* self :angle-vector av tm :ctype :arm-controller args))
```

Comment:
I think `:angle-vector-motion-plan` is irregular so that having controller-type arg as `&optional` not as `&key` in `:angle-vector-motion-plan` seems make sense for me.as `&optional` not as `&key` 
That API is not used for 20 years (maybe less than 5 years).
as `&optional` not as `&key` 
Both `&optional` and `&key` have pros and cons,as `&optional` not as `&key`  and I do not think `&optional` args in `:angle-vector` should be `&key`.as `&optional` not as `&key` 

I think the biggest problem for beginners is documentation.